README.md

```diff
@@ -60,79 +60,78 @@ It offers:
 In the age of Foundation Models, control over your data is key and building pipelines
 for large-scale data processing is costly, especially when they require advanced
 machine learning-based operations. This need not be the case, however, if processing
-components would be reusable and exchangeable and pipelines were easily composable. 
+components would be reusable and exchangeable and pipelines were easily composable.
 Realizing this is the main vision behind Fondant.
 
-<p align="right">(<a href="#chocolate_bar-fondant">back to top</a>)</p>
+<p align="right">(<a href="">back to top</a>)</p>
```

Comment:
This will reload the page. You can:
- Add an invisible anchor at the top of the page:
  `<a name="top"></a>`
- Reference it here:
  ```suggestion
  <p align="right">(<a href="top">back to top</a>)</p>
  ```

Reply:
Not sure of doing it well but it doesn't work for me. It just sends me to another page with 404 error, so it doesn't actually keep me on the same page.

Reply:
On Github or on the docs website? I tested it on Github and it worked.

Reply:
On the docs website.